capstone/src/main/java/com/google/sps/servlets/announcements/ScheduleAnnouncementServlet.java

```diff
@@ -0,0 +1,93 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example club content */
+@WebServlet("/schedule-announcement")
+public class ScheduleAnnouncementServlet extends HttpServlet {
+
+  private ZoneId timeZone = ZoneId.of("America/Los_Angeles");
+  private Clock clock = Clock.system(timeZone);
+
+  protected void setClock(Clock clock) {
+    this.clock = clock;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final String userEmail = request.getUserPrincipal().getName();
+    final String clubName = request.getParameter(Constants.PROPERTY_NAME);
+    final String announcementContent = request.getParameter(Constants.CONTENT_PROP);
+    String scheduledDate = request.getParameter(Constants.SCHEDULED_DATE_PROP);
+    if (scheduledDate.charAt(scheduledDate.length() - 1) != 'Z') {
```

Comment:
I think String has an endsWith() method that might be helpful here. https://www.tutorialspoint.com/java/java_string_endswith.htm